homeassistant/helpers/config_validation.py

```diff
@@ -80,9 +80,10 @@ def isfile(value: Any) -> str:
     return file_in
 
 
-def ensure_list(value: Union[T, Sequence[T]]) -> List[T]:
+def ensure_list(value: Union[T, Sequence[T]]) -> Sequence[T]:
     """Wrap value in list if it is not one."""
-    return value if isinstance(value, list) else [value]
+    return (value if isinstance(value, list) else
```

Comment:
Ok, will roll back this change.
Ok if I comment out the failing unit tests with `[None]` with a comment linking to the voluptuous PR?